hazelcast-command-line/src/main/java/com/hazelcast/commandline/AbstractCommandLine.java

```diff
@@ -0,0 +1,79 @@
+/*
+ * Copyright (c) 2008-2021, Hazelcast, Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.hazelcast.commandline;
+
+import picocli.CommandLine;
+
+import java.io.PrintStream;
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+import java.util.Stack;
+
+/**
+ * Abstract command line class. The methods and properties in this class are shared in other commandline implementations.
+ */
+abstract class AbstractCommandLine implements Runnable {
```

Comment:
There were 2 subclasses for this abstract class, one of them for the `mc` command, I think we can remove this now.

Reply:
AbstractCommandLine has been inlined in https://github.com/hazelcast/hazelcast/pull/20262/commits/e6abc9a4febc078ebfda8e5f4e44694556e21e86